### PR TITLE
New version of bubble chart

### PR DESCRIPTION
**Pull request by digrich**
Please release the new version of bubble chart. It's having couple of bug fixes along with grafana > 6.6 compatibility.

### CODE REVIEW

File: repo.json [hunk 1]
```diff
@@ -2415,6 +2415,11 @@
           "version": "1.1.0",
           "commit": "0d2ee17f7dd7516affec5b1eca502d1bc1e2d212",
           "url": "https://github.com/digrich/bubblechart-panel"
+        },
+        {
+          "version": "1.2.0",
```
Comment (daniellee) [inline repo.json:2420]: The version in the [plugin.json file](https://github.com/digrich/bubblechart-panel/blob/e86d545dcbb3cf63b4233ffb056de58e7f102c4d/dist/plugin.json#L25) needs to be updated to 1.2.0,

Comment (digrich) [reply repo.json:2420]: Updated.

**Comment by daniellee** [comment]
@digrich Grafana loads the plugin.json file from the plugin's dist folder: https://github.com/digrich/bubblechart-panel/blob/297ebe0a171661086e25fca0f6afad9e49c4758a/dist/plugin.json#L25 so the version number is still incorrect.

**Comment by digrich** [comment]
Hi @daniellee - Sorry about that. I have updated dist folder with latest changes.

**Comment by digrich** [comment]
Hi @daniellee  - Are we enforcing any signature verification of plugins while publishing the updates?

**Comment by marcusolsson** [comment]
Hi @digrich! Could you fix the issues reported by [this plugin validator](https://grafana-plugins-web-vgmmyppaka-lz.a.run.app/)?

Copy-paste the following in the text field, and press Enter:

```
digrich/bubblechart-panel@df8a4129bb4298f5d200696621a7e13ed9af9c01
```

**Comment by digrich** [comment]
Hi @marcusolsson -  I have fixed the issues reported by plugin validator. Can you please take a look and publish the plugin?

**Comment by marcusolsson** [comment]
Thank you! You can find the published plugin here:

https://grafana.com/grafana/plugins/digrich-bubblechart-panel?version=1.2.0

**Pull request merged**